src/main/java/bral/museum/ApiKey.java

```diff
@@ -0,0 +1,38 @@
+package bral.museum;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Retrieve the API Key in apikey.properties or from environment variable on GitHub
+ */
+public class ApiKey {
```

Comment:
The new instructions that was sent with Part 1 has instructions adding this class as a dependency, not as a source file.